pkg/cloudprovider/suite_test.go

```diff
@@ -147,6 +148,38 @@ var _ = Describe("CloudProvider", func() {
 							Zone: "test-zone-1c",
 						},
 					},
+					AMIs: []v1beta1.AMI{
+						{
+							ID: "ami-test1",
+							Requirements: scheduling.NewRequirements(
```

Comment:
Still wondering why we're not using the NewNodeSelectorRequirements here

Reply:
Updated to include that